### DISCUSSION TITLE
Can not connect to remote via ssh on windows

### DISCUSSION THREAD

**Discussion by flemming-pr** [spatie/ray#566] [Q&A]
I want to connect to a remote server via ray(1.17.0.0) for windows. I have changed the private key path to `C:\Users\myuser\.ssh\` but I always get the following error:
`Error: Error invoking remote method  connectSsh : Error EISDIR: illigal operation on directory, read`

Does anybody have a solution?

**Comment by AdrianMrn** [comment]
I think the problem is that your private key path needs to point to the file containing the private key, not the directory.

**Reply by flemming-pr** [reply]
I have pointed the path to the direct file.
![image](https://user-images.githubusercontent.com/7942983/146894546-daf4bb2e-ba30-4c12-b8cc-c881ccaf7634.png)
